stdlib/public/core/ClosedRange.swift

```diff
@@ -488,3 +488,26 @@ extension ClosedRange {
 // shorthand. TODO: Add documentation
 public typealias CountableClosedRange<Bound: Strideable> = ClosedRange<Bound>
   where Bound.Stride : SignedInteger
+
+extension ClosedRange: Codable where Bound: Codable {
+  private enum CodingKeys: String, CodingKey {
+    case lowerBound = "from"
+    case upperBound = "to"
+  }
+
+  public init(from decoder: Decoder) throws {
+    let container = try decoder.container(keyedBy: CodingKeys.self)
+    let lowerBound = try container.decode(Bound.self, forKey: .lowerBound)
+    let upperBound = try container.decode(Bound.self, forKey: .upperBound)
+    guard lowerBound <= upperBound else {
+      throw DecodingError.dataCorruptedError(forKey: CodingKeys.upperBound, in: container, debugDescription: "upperBound (to) cannot be lower than lowerBound (from)")
```

Comment:
Nit: the Swift standard library wraps to 80-character lines.

Reply:
`Codable.swift` originated as part of Foundation and was later moved into the standard library, so it has slightly different conventions, but that's a unique case. If a string is too long, it can overflow the limit.